buildscripts/linux-static/build-deps.sh

```diff
@@ -22,12 +22,36 @@ pushd "$ROOT_SRC"
       git submodule update --init qtbase qtdeclarative qtgraphicaleffects qtimageformats qtquickcontrols qtsvg qtx11extras qttools qtmultimedia
       git submodule foreach --recursive git reset --hard
       git reset --hard
+      # TODO:
+      # Find which configure options can be included to compile QT without breaking the compilation
```

Comment:
this is great

Reply:
@pospeselr are you happy for me to merge this on master? Just wanted to check that isn't going to create any conflicts for you.

Reply:
Yeah it's good, doesn't block me at all